src/vehicle.h

```diff
@@ -1448,7 +1448,10 @@ class vehicle
         // id of the om_vehicle struct corresponding to this vehicle
         int om_id;
         // direction, to which vehicle is turning (player control). will rotate frame on next move
+        // must be a multiple of 15 degrees
         int turn_dir = 0;
+        // actual direction in degrees
+        int actual_turn_dir = 0;
```

Comment:
Why do we need this separate member? Can`t you just use `turn_dir` directly? (Only add the rounding code at the end of the PR, but use `turn_dir` as input.)

Reply:
I'm not sure what I was thinking here.  Removed.